src/Microsoft.Dnx.Compilation.CSharp/RoslynCompiler.cs

```diff
@@ -355,7 +327,7 @@ private static string RemovePrereleaseTag(string version)
 
             foreach (var d in dirs)
             {
-                ctx.Monitor(new FileWriteTimeCacheDependency(d));
+                ctx?.Monitor(new FileWriteTimeCacheDependency(d));
```

Comment:
It is already :) I made that change because the first time I ran my test, it blew up there with a NullReferenceException.